spec.yml

```diff
@@ -258,6 +258,29 @@ paths:
         '403':
           $ref: '#/components/responses/Forbidden'
 
+  /partner/product-families:
+    get:
+      tags:
+        - product-families
+      summary: Get all the available product families, their products, and prices.
```

Comment:
Should we perhaps add the partner specificity to the summary?
E.g.: `Get all product families, their products, and prices available to a specific partner.`